src/mdViews/patientOverview/StatusComponents/PatientInformationWidget.tsx

```diff
@@ -1,28 +1,135 @@
-import React, { FC } from 'react';
+import React, { FC, useEffect, useState } from 'react';
 import {
+  Text,
   Flex,
   useBoolean,
-  useColorModeValue,
   FormLabel,
   FormControl,
   Editable,
   EditablePreview,
   EditableInput,
+  Divider,
+  Box,
 } from '@chakra-ui/react';
-import { Patient } from '../../../core/models/Patient';
 import { useMapper } from '../../../core/services/resourceMapper/ResourceMapperContext';
 import { useParams } from 'react-router-dom';
+import Select from 'react-select';
+import { convertArrayToOptionArray } from '../../settings/vaccineInformationCard';
+import { Country } from 'country-state-city';
+import { getStatesOfCountry } from 'country-state-city/dist/lib/state';
+import { Gender } from '../../../core/models';
+import { CloseIcon } from '@chakra-ui/icons';
+import { FaWrench } from 'react-icons/fa';
+import { ICountry, IState } from 'country-state-city/dist/lib/interface';
+import { usePatients } from '../../../hooks';
+import { cloneDeep } from 'lodash';
 
 export const PatientInformationWidget: FC = ({}) => {
-  const mapper = useMapper();
   const params = useParams();
   const [editMode, setEditMode] = useBoolean(false);
-  const patient: Patient | undefined = mapper.getPatientById(
-    params['patientId'] || ''
+  const [initialized, setInitialized] = useBoolean(false);
+
+  const allCountries = Country.getAllCountries();
+  const allCountryNames = allCountries.map(
+    (potentialCountry) => potentialCountry.name
   );
+
+  const { patients } = usePatients({ _id: params['patientId']! });
+  const patient = patients?.at(0);
```

Comment:
It would be more efficient here to use the `getById` query directly:
```suggestion
  const { data: patientRaw } = patientApi.endpoints.getById.useQuery(params['patientId']!);
  const patient = PatientMapper.fromResource(patientRaw);
```
This also prevents this component from reloading if someone creates a patient. The `.get` endpoint expects to be used for lists of patients, so it must reload if the FHIR Server sends us a notification of a `create` event for a `Patient` resource. This is because the new patient could be included in the list returned by the FHIR server.